training/tf/chunkparser.py

```diff
@@ -167,14 +165,23 @@ def convert_v3_to_tuple(self, content):
         # Enforce move_count to 0
         move_count = 0
 
-        # Unpack bit planes
-        planes = np.unpackbits(np.frombuffer(planes, dtype=np.uint8))
+        # Unpack bit planes and cast to 32 bit float
+        planes = np.unpackbits(np.frombuffer(planes, dtype=np.uint8)).astype(np.float32)
+        rule50_plane = (np.zeros(8*8, dtype=np.float32) + rule50_count) / 100
```

Comment:
Is it usual in NN to normalize by highest value? (and actually, highest value would be 99, not 100).
Maybe it makes sense to normalize by lower value, as most of the time value of this plane would be low.

In general I guess proper normalization is dividing all planes by their average expected stddev, but that's out of the scope of this PR for sure.

Reply:
Ideally we'd apply a whitening transformation, but we don't know the true distribution. I'm also not sure it matters a lot...